index.html

```diff
@@ -223,52 +141,84 @@ <h3>Skills</h3>
     <section class="section-contact" id="contact-section">
       <div class="desktop-contact d-flex">
         <p class="contact-desc">
-          I'm always interested in hearing about new projects, so if you'd like to
-          chat please get in touch.
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
         </p>
-        <form class="contact-form" action= "https://formspree.io/f/xqkoyalw" method="post">
+        <form
+          class="contact-form"
+          id="form"
+          action="https://formspree.io/f/xqkoyalw"
+          method="post"
+        >
           <label for="firstname">First name: </label>
-          <input name="name" maxlength="30" type="text" id="name" placeholder="Full name" required />
+          <span id="fullNameError"></span>
+          <input
+            name="name"
+            maxlength="30"
+            type="text"
+            id="name"
+            placeholder="Full name"
+          />
           <label for="email">Email: </label>
```

Comment:
- [ ] Your form input elements are not aligned like in the figma design. Check screenshots below:
- Your design
![Screenshot from 2023-04-14 23-22-38](https://user-images.githubusercontent.com/62849872/232164968-2c8bb9ce-5385-40ea-a0c5-991b0b01c28f.png)

- Figma design
![image](https://user-images.githubusercontent.com/62849872/232164930-8f6df02e-d514-437d-bf25-1bf5294ce767.png)